tests/components/test_smoothing.cpp

```diff
@@ -50,36 +56,35 @@ TEST_CASE("smoothing_Newton_Method")
     CHECK((uv1 - uv2).norm() - (uv0 - uv2).norm() < 1e-6);
 }
 
-TEST_CASE("smoothing_Newton_Method_line_search")
-{
-    DEBUG_TriMesh mesh = ten_triangles_with_position(2);
-    OperationSettings<tri_mesh::VertexSmoothUsingDifferentiableEnergy> op_settings;
-    op_settings.coordinate_handle =
-        mesh.get_attribute_handle<double>("position", PrimitiveType::Vertex);
-    op_settings.smooth_boundary = false;
-    op_settings.second_order = true;
-    op_settings.line_search = true;
-    op_settings.energy = std::make_unique<function::AMIPS2D>(mesh, op_settings.coordinate_handle);
-    op_settings.initialize_invariants(mesh);
-    Scheduler scheduler(mesh);
-    const auto& factory =
-        scheduler.add_operation_type<operations::tri_mesh::VertexSmoothUsingDifferentiableEnergy>(
-            "optimize_vertices",
-            std::move(op_settings));
-    Tuple tuple = mesh.face_tuple_from_vids(2, 4, 5);
-    spdlog::warn(
-        "Initial gradient: norm: {}",
-        factory.settings().energy->get_one_ring_gradient(tuple).norm());
-    while (factory.settings().energy->get_one_ring_gradient(tuple).norm() > 1e-10) {
-        scheduler.run_operation_on_all(PrimitiveType::Vertex, "optimize_vertices");
-        tuple = mesh.face_tuple_from_vids(2, 4, 5);
-    }
-    ConstAccessor<double> pos = mesh.create_const_accessor(op_settings.coordinate_handle);
+// TEST_CASE("smoothing_Newton_Method_line_search")
```

Comment:
If that test case is outdated, please remove it. If there is a good reason to keep that commented out code, please add a comment to it so that we know it should not be deleted.